tests/models/mapped_archesjson_import_tests.py

```diff
@@ -33,6 +33,12 @@
 
 
 class mappedArchesJSONImportTests(ArchesTestCase):
+    @classmethod
+    def setUpClass(self):
```

Comment:
what's up with changing the test logic for this branch?

Reply:
Right. I was tempted to break this out into its own PR but didn't make it that far. 😅 

When first adding tests I was getting a cryptic failure like this from unrelated tests. [Here's a CI job that failed](https://github.com/archesproject/arches/actions/runs/7674762947/job/20919934517):

```
======================================================================
ERROR: setUpClass (tests.permissions.permission_tests.PermissionTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.12.1/x64/lib/python3.12/site-packages/django/db/backends/utils.py", line 87, in _execute
    return self.cursor.execute(sql)
           ^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.ObjectInUse: cannot TRUNCATE "geojson_geometries" because it has pending trigger events
CONTEXT:  SQL statement "TRUNCATE TABLE geojson_geometries"
PL/pgSQL function refresh_geojson_geometries() line 3 at SQL statement
```

You can reproduce locally by reverting 840ee8d118d01b8947508c235a5d6a3e581061ac and just running a subset of tests, e.g. `python3 manage.py test tests.views --settings=tests.test_settings`

I think what happened is that my new test class uses `setUpTestData()` to create the same crop of objects before each test and tear the down before the next test without the need to write any additional cleanup code. Django implements that via transactions. Then, one of the tests actually tests an error condition were a transaction inside the view I wrote has to abort.

So, somewhere in the midst of that two layers of transactions aborting, the test runner went to the next class, executed the setup code, which called out to an arches CLI command, which tripped on not being able to call a raw database function because one or more transactions was in the middle of being rolled back. Something like that.

I think that's because these (and similar) no-op overrides were "silencing" Django's built in transaction cleanup:

https://github.com/archesproject/arches/blob/48636759917c9346bcfc8beeccdf540b77e48683/tests/base_test.py#L132-L136

Removing the no-op overrides made the cryptic errors go away, but as you can imagine, caused more trouble because now that more cleanup was happening, it exposed undeclared dependencies among test classes, like this one you've highlighted, where tiles are being inserted in setup code, but the graphs may not have been created (e.g. they were only created before by accident of the order the tests ran in).

If you want I can pull this out into a separate PR and target 7.6 with it. LMK

Reply:
Pulled out for separate review in #10711